lib/matplotlib/style/core.py

```diff
@@ -55,7 +55,7 @@ def _remove_blacklisted_style_params(d, warn=True):
             if warn:
                 _api.warn_external(
                     "Style includes a parameter, '{0}', that is not related "
-                    "to style.  Ignoring".format(key))
+                    "to style.  Ignoring this parameter.".format(key))
```

Comment:
Change to an f-string while you're here:
```suggestion
                    f"Style includes a parameter, {key!r}, that is not related "
                    f"to style.  Ignoring this parameter.")
```

Reply:
Done